src/Components/ClusterRules/ClusterRules.spec.ct.js

```diff
@@ -40,45 +40,29 @@ const filtersConf = {
     selectorText: 'Description',
     values: ['Lorem IPSUM', '1Lorem', 'Not existing recommendation'],
     type: 'input',
+    f: (it, value) =>
+      it.description.toLowerCase().includes(value.toLowerCase()),
```

Comment:
maybe it is worth abstracting `it` from the rule content, and assuming that it already contains a value that we want to compare.

for example, here, and in other places, you would use just `it` instead of looking for particular params like `it.total_risk`, `it.tags` and etc.

Reply:
I do not understand. Different `f` will need to access different parameters as you pointed out. How would you handle that?